docs/source/array-design.rst

```diff
@@ -105,6 +105,32 @@ Some ways in which ``chunks`` reflects properties of our array:
        ('x', 1, 0), ('x', 1, 1), ('x', 1, 2)
 
 
+Metadata
+--------
+
+Many Array operations rely on knowing the name and dtype of columns.  To
```

Comment:
Unlike dataframse, arrays don't have columns, so the meta keeps track of `dtype` (int, float) and `type` (numpy, cupy, ...)

Reply:
I have updated the description as you suggested.